src/index.js

```diff
@@ -20,6 +20,200 @@ import {
   failingContractBytecode,
 } from './constants.json';
 
+function findElementById(id) {
+  const element = document.getElementById(id);
+
+  if (element === null) {
+    throw new Error(`Couldn't find element by id '${id}'`);
+  } else {
+    return element;
+  }
+}
+
+function setupFilters(filterType, filterChangesBoxPlaceholder) {
+  if (!['log', 'block', 'pendingTransaction'].includes(filterType)) {
+    throw new Error(
+      "filterType must be either 'log', 'block', or 'pendingTransaction'",
+    );
```

Comment:
Mostly to get rid of double-quotes ;)
```suggestion
  const validFilterTypes = ['log', 'block', 'pendingTransaction'];
  if (!validFilterTypes.includes(filterType)) {
    throw new Error(
      `filterType must be one of: ${validFilterTypes.join(',')}.`,
    );
```